requirements/edx/base.txt

```diff
@@ -4,6 +4,8 @@
 #
 #    make upgrade
 #
+-e git+https://github.com/openedx/blockstore.git@1.2.1#egg=blockstore==1.2.1
+    # via -r requirements/edx/github.in
```

Comment:
From the pypi page, it looks like we haven't had a release since May 24 2016, so I'm not sure what needs to be done to kickstart those releases again.

@bradenmacdonald  or @symbolist , maybe you know?

Reply:
Ah lol, that's not our blockstore at all, that's https://github.com/stacks-network/stacks-blockchain

So we'd need to organize our own openedx-blockstore package. Pings to Braden and Usman are still relevant though, as core committers on blockstore.